client.go

```diff
@@ -49,9 +49,9 @@ var (
 const (
 	defaultTCPAddress  = "127.0.0.1:6640"
 	defaultUnixAddress = "/var/run/openvswitch/ovnnb_db.sock"
-	SSL             = "ssl"
```

Comment:
why you unexport this const ?

Reply:
gofmt wasn't happy with it all CAPS. I couldn't find it used outside of project so made local. I can revert and add comments for extern variables/const if intended to be externed.